draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -1048,7 +1101,7 @@ notifications.
 ## Optical Topology Discovery
 
 The WSON Topology Model and the Flexi-grid Topology model can be used
-to report the DWDM network topology (e.g., WDM nodes and OMS links),
+to report the DWDM network topology (e.g., DWDM nodes and OMS links),
```

Comment:
I-D.ietf-ccamp-optical-impairment-topology-yang defines the 'WDM node' (and not the 'DWDM node')

```suggestion
to report the DWDM network topology (e.g., WDM nodes and OMS links),
```